pkg/promtail/client/client.go

```diff
@@ -60,6 +60,11 @@ var (
 		Name:      "dropped_entries_total",
 		Help:      "Number of log entries dropped because failed to be sent to the ingester after all retries.",
 	}, []string{"host"})
+
+	countersWithHost = []*prometheus.CounterVec{
```

Comment:
I would just move this after `requestDuration` definition, to keep all metrics definition grouped together

Reply:
Sounds good

Reply:
Amended existing commit with the requested change.